proto/agent.go

```diff
@@ -232,17 +237,20 @@ func (a *Agent) finishTask(ctx context.Context, tc *taskContext, status string,
 	return resp, nil
 }
 
-func (a *Agent) endTaskResponse(tc *taskContext, status string, timeout bool) *apimodels.TaskEndDetail {
-	detail := &apimodels.TaskEndDetail{
+func (a *Agent) getCurrentCommandType(tc *taskContext) string {
+	if tc.taskConfig != nil {
+		return tc.currentCommand.GetType(tc.taskConfig.Project)
+	}
+	return ""
```

Comment:
Should this line return the default instead?

Reply:
done